gprofiler/metadata/__init__.py

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
+from typing import Any, Dict
+
+# our metadata is JSON-serializable dicts that begin with string keys.
+Metadata = Dict[str, Any]
```

Comment:
you introduce this while we do have [`Metadata` in granulate-utils](https://github.com/Granulate/granulate-utils/blob/d99905de5a06808db664f3af2ac83842119a106e/granulate_utils/metadata/__init__.py#L8C38-L8C42)
why do we need both?
we should either:
1. "merge" them into one
2. if there's a reason for both of them to exist, at the very least make naming more distinctive.

there is a possibility these `Metadata` stuff needs to be treated in a different PR.

Reply:
I actually did merge them into one initially, but ended up separating them. They are not related and mean different things, no need to mix up the imports as IMO it only causes confusion.

I agree that naming them more distinctively will help - I'll do that

Reply:
Renamed to ProfileMetadata, together with the comment I think it's good enough.